docs/UserGuide.md

```diff
@@ -145,28 +70,95 @@ as a link to your contact at index 2.`
 
 ![add link success](images/add-remark/add-link.png)
 
-### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
 
-Format: `list`
+### **Assign priority levels:** **`add /priority /index`** 
```

Comment:
This line should not have bold formatting.
The text would alr have been bolded by the hashes.